lib/web/fetch/constants.js

```diff
@@ -22,10 +22,9 @@ const badPorts = /** @type {const} */ ([
 const badPortsSet = new Set(badPorts)
 
 /**
- * @see https://w3c.github.io/webappsec-referrer-policy/#referrer-policies
+ * @see https://w3c.github.io/webappsec-referrer-policy/#referrer-policy-header
  */
-const referrerPolicy = /** @type {const} */ ([
-  '',
+const referrerPolicyTokens = /** @type {const} */ ([
```

Comment:
We use this set of values for determining the right policy. If an empty string would be in this set, it could result in a wrong policy, if the header is e.g. `"no-referrer, same-origin, "`

Reply:
```js
// 3. For each token in policy-tokens, if token is a referrer policy and token is not the empty string, then set policy to token.
```

Reply:
can you add a test for that?